tox.ini

```diff
@@ -102,3 +102,12 @@ deps =
     -r{toxinidir}/requirements/js_test.txt
 commands =
     jasmine-ci
+
+[testenv:pii-annotations]
+setenv =
+    DJANGO_SETTINGS_MODULE = enterprise.settings.test
+deps =
+    -r{toxinidir}/requirements/test-master.txt
+commands =
+    code_annotations django_find_annotations --config_file .pii_annotations.yml --report_path pii_report/ \
```

Comment:
Maybe leave out `--report_path`? Should be set to same path via the config file.